src/main/java/me/jellysquid/mods/sodium/client/gl/state/GlStateTracker.java

```diff
@@ -3,8 +3,8 @@
 import me.jellysquid.mods.sodium.client.gl.array.GlVertexArray;
 import me.jellysquid.mods.sodium.client.gl.buffer.GlBuffer;
 import me.jellysquid.mods.sodium.client.gl.buffer.GlBufferTarget;
-import org.lwjgl.opengl.GL15;
-import org.lwjgl.opengl.GL30;
+import me.jellysquid.mods.sodium.client.gl.func.GlFunctions;
+import org.lwjgl.opengl.*;
```

Comment:
replace the wildcard with the .GL15/GL30 to match gtnh style, even though spotless is exploding

Reply:
Yes - please remove the star imports.